### DISCUSSION TITLE
Fechar Pull Requests sem interações por muito tempo

### DISCUSSION THREAD

**Discussion by AllanSiqueira** [campuscode/rails-guides-pt-BR#831] [Geral]
Pessoal, vi que temos muitos PRs que estão bastante tempo sem interação, o que acha de termos algum guia no guia de contribuição.

Falo isso pois com vários PRs abertos antigos, pode parecer que este projeto não é mais mantido, o que acham?

**Comment by thais-kusuki-vindi** [comment]
Pra mim, faz sentido @AllanSiqueira
